raiden-ts/CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # Changelog
 
 ## [Unreleased]
+### Changed
+- [#2049] Target ES2019 (NodeJS 12+) on SDK builds
```

Comment:
Do we have some signal/flag/hint/... that this is a breaking change? :thinking:

Reply:
I don't think we have. At least, never went through it. In any case, this shouldn't be "breaking" right now, in the sense the the SDK and CLI already implicitly required Node12, so it shouldn't break any current usecase.